webpack.config.js

```diff
@@ -75,7 +75,9 @@ const babelLoader = {
 	},
 };
 
-module.exports = function( env, argv ) {
+// The extensionName arg here is set by bin/sdk/gutenberg.js when building Gutenberg
+// extensions.
+module.exports = function( { extensionName = '' } = {}, argv ) {
```

Comment:
can we rearrange the newlines here and inline the docs? I feel like as we add more the comments will get further away from the code

```js
module.exports = function( {
	extensionName = '', // set by SDK builder scripts as a general sub-project name
} = {}, args ) {
```